Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:16-alpine
+FROM node:16-alpine as dashboard-builder
```

Comment:
is this right? is this the dashboard builder?

Reply:
Yes